Flashcards.DataAccess/Repositories/DeckRepository.cs

```diff
@@ -173,5 +175,58 @@ public Deck Update(Deck deck)
 
             return deck;
         }
+
+        public Deck CreateCopied(Deck deck)
```

Comment:
code smell but idk the nice way to make it shorter
https://refactoring.guru/smells/long-method

Reply:
I know, it would be 4 lines if we only didn't have to change from models to entities all the time... I guess we could write a converter